### PR TITLE
Razvan/layers

### PR DESCRIPTION
**Pull request by razvanch**

### CODE REVIEW

**Comment by sils** [comment]
ack https://github.com/coala/docker-coala-base/pull/58/commits/8d9f9ff687015c20779dc5081409a96be6876468
https://github.com/coala/docker-coala-base/pull/58/commits/ba0a5dc855a77c35e7569986c0e1454115997f54

**Comment by sils** [comment]
ack https://github.com/coala/docker-coala-base/pull/58/commits/8d9f9ff687015c20779dc5081409a96be6876468

**Comment by sils** [comment]
ack https://github.com/coala/docker-coala-base/pull/58/commits/c0ad95451c38ac1be941e53d5195e4813f11009e

**Comment by sils** [comment]
@rultor merge

File: Dockerfile [hunk 1]
```diff
@@ -2,93 +2,80 @@ FROM opensuse:tumbleweed
...
+  zypper addrepo -f \
+    http://download.opensuse.org/repositories/devel:/languages:/lua/openSUSE_Factory/ \
+    devel:languages:lua && \
+  # Use Leap for nodejs
+  zypper addrepo http://download.opensuse.org/repositories/devel:languages:nodejs/openSUSE_Leap_42.2/devel:languages:nodejs.repo && \
+  # Add repo for rubygem-bundler
+  zypper addrepo http://download.opensuse.org/repositories/home:AtastaChloeD:ChiliProject/openSUSE_Factory/home:AtastaChloeD:ChiliProject.repo && \
+  # Package dependencies
+  zypper --no-gpg-checks --non-interactive dist-upgrade && \
+    zypper --non-interactive install -t pattern devel_basis && \
```
Comment (jayvdb) [inline Dockerfile:20]: why is this indented an extra level ?  This broke `git blame` for all of the package list below.

**Pull request merged**